apps/studio/components/ui/AIAssistantPanel/AIAssistant.tsx

```diff
@@ -221,6 +226,64 @@ export const AIAssistant = ({ className }: AIAssistantProps) => {
     [snap]
   )
 
+  const editMessage = useCallback(
+    (messageId: string) => {
+      const messageIndex = chatMessages.findIndex((msg) => msg.id === messageId)
+      if (messageIndex === -1) return
+
+      // Target message
+      const messageToEdit = chatMessages[messageIndex]
+
+      // Activate editing mode
+      setEditingMessageId(messageId)
+      setOriginalMessageContent(messageToEdit.content ?? '')
```

Comment:
What do we use originalMessageContent for? Is it needed?

Reply:
thanks @saxonf, yeah, you're right, we're already passing `value`, so `originalMessageContent` is unnecessary.

fixed it! reference: [`290db4d` (#37603)](https://github.com/supabase/supabase/pull/37603/commits/290db4de2043f88d9b1597d2ca8ce61a22e19611)